StellarDotnetSdk.Tests/SorobanServerTest.cs

```diff
@@ -565,9 +567,8 @@ private async Task<SimulateTransactionResponse> SimulateAndUpdateTransaction(Tra
 
         tx.SetSorobanTransactionData(simulateResponse.SorobanTransactionData);
         if (simulateResponse.SorobanAuthorization != null)
-        {
```

Comment:
I just added these before in a comment please use the brackets at all time.

Reply:
Ah, the Rider removed it when I ran Reformat and Cleanup. What should we do?